src/main/java/com/backbase/ct/bbfuel/configurator/TransactionsConfigurator.java

```diff
@@ -53,6 +53,10 @@ public void ingestTransactionsByArrangement(String externalArrangementId, boolea
         List<SubCategory> finalCategories = new ArrayList<>(retailCategories);
 
         if (isRetail) {
+            // Add 1 check images per account.
+            transactions.add(reader.loadSingleWithCheckImages(externalArrangementId));
+
+            // After that ingest rest of the transactions.
```

Comment:
`loadSingle` only loads one transaction, so "....rest of the transactions." is not literally correct

Reply:
actually it does iterates minimum 10 times, according to the given randomNumber.